src/Data/List/Relation/Binary/Permutation/Propositional/Properties.agda

```diff
@@ -218,138 +176,34 @@ drop-mid {A = A} {x} ws xs p = drop-mid′ p ws xs refl refl
       _ ∷ _ ∷ xs ++ _   ∎
   drop-mid′ (swap y z p) (y ∷ z ∷ ws) (z ∷ [])     refl refl = begin
       _ ∷ _ ∷ ws ++ _   <<⟨ refl ⟩
+-- *** NB the next step can't be replaced with <⟨ shift _ _ _ ⟨ for some reason? ***
       _ ∷ (_ ∷ ws ++ _) <⟨ ↭-sym (shift _ _ _) ⟩
+-- *** because of parsing problems for that use of the `Reasoning` combinators!? ***
```

Comment:
This seems to be a rare bug in our `Reasoning` combinators:
* `<⟨ ↭-sym (shift _ _ _) ⟩` and
* `<⟨ shift _ _ _ ⟨`
*ought to have the same semantics here; the first succeeds, but the second fails with a parsing error. Seemingly on `master` as well, so will flag as an issue cf. #2319